lib/elixir/lib/io.ex

```diff
@@ -1,18 +1,18 @@
 defmodule IO do
   @moduledoc """
-  Functions handling IO.
+  Functions handling input/output (IO).
 
   Many functions in this module expect an IO device as an argument.
   An IO device must be a pid or an atom representing a process.
   For convenience, Elixir provides `:stdio` and `:stderr` as
   shortcuts to Erlang's `:standard_io` and `:standard_error`.
 
-  The majority of the functions expect char data, i.e. strings or
+  The majority of the functions expect chardata, i.e. strings or
   lists of characters and strings. In case another type is given,
   functions will convert to string via the `String.Chars` protocol
   (as shown in typespecs).
 
-  The functions starting with `bin*` expect iodata as an argument,
+  The functions starting with "bin" expect iodata as an argument,
```

Comment:
Good point. I'd leave the backticks then, and let someone else decide what to do with the `*` 😄